ckanext/data_qld/reporting/constants.py

```diff
@@ -8,3 +8,6 @@
     REPORT_TYPE_ENGAGEMENT,
     REPORT_TYPE_ADMIN
 ]
+
+REPORT_DEIDENTIFIED_NO_SCHEMA_COUNT_FROM = "ckanext.data_qld.reporting.de_identified_no_schema.count_from"
+REPORT_DEIDENTIFIED_NO_SCHEMA_COUNT_FROM_DF = "2022-01-01"
```

Comment:
What is this? "DF" is cryptic and a hard-coded date is a yellow flag.

Reply:
Removed the value from the constants, moved to `project.ini`.

Reply:
Will be fixed and pushed by Alex shortly.